agreement_rebate/tests/test_agreement_rebate.py

```diff
@@ -112,6 +111,7 @@ def setUpClass(cls):
         )
 
     @classmethod
+    @freeze_time("2022-02-01 09:30:00")
```

Comment:
Why do you need it in these auxiliary methods? Isn't only needed in the `def test_` ones?

Reply:
It is necessary to define the decorator there because it is used in the setup() of the https://github.com/OCA/agreement/blob/67e75e6c85abfb47db5778fa99c2ee3dcafe55ea/agreement_rebate/tests/test_agreement_rebate.py#L104 test.

Another option is to define in https://github.com/OCA/agreement/blob/67e75e6c85abfb47db5778fa99c2ee3dcafe55ea/agreement_rebate/tests/test_agreement_rebate.py#L121 directly the specific date (2022-01-01).

Reply:
I prefer the second option then. What I was referring anyway is if the decorator is not "propagated" from the first method to the following called ones.

Reply:
Changes done.